components/input/navigationRectangle.js

```diff
@@ -1,130 +1,100 @@
 import { View } from "react-native";
-import { Icon, Text } from "react-native-paper";
+import { Divider, Icon, Text } from "react-native-paper";
 
 import { getIconByKey } from "~/Utility";
 
-function requirementDisplay(attemptShots, drillInfo, shotIndex) {
-  for (let i = 0; i < drillInfo.requirements.length; i++) {
-    const requirementName = drillInfo.requirements[i].name;
-    //Target Requirement takes priority in multi-requirement Drills
-    if (requirementName == "target") {
-      return (
-        <Text
-          style={{
-            fontWeight: "bold",
-            fontSize: 16,
-            padding: 2,
-          }}
-        >
-          {drillInfo.requirements[0].prompt}:{" "}
-          {attemptShots[shotIndex - 1].items.target}{" "}
-          {drillInfo.requirements[0].distanceMeasure}
-        </Text>
-      );
-    }
-    //Club Requirement takes next priority in multi-requirement Drills
-    if (requirementName == "club") {
-      return (
-        <Text
-          style={{
-            fontWeight: "bold",
-            fontSize: 16,
-            padding: 2,
-          }}
-        >
-          {drillInfo.requirements[0].prompt}:{" "}
-          {attemptShots[shotIndex - 1].items.club}
-        </Text>
-      );
-    }
-  }
-
-  //Return empty if no requirement display is found
-  console.log(
-    "Requirement Display error in requirementDisplay located within navitgationRectangle component",
-  );
-  return <></>;
-}
-
-export default function NavigationRectangle({
-  drillInfo,
-  attemptShots,
-  inputValues,
-  shotIndex,
-}) {
+export default function NavigationRectangle({ drillInfo, inputValues, shot }) {
+  console.log("inputValues", inputValues);
+  const keys = Object.keys(inputValues);
   return (
     <View
       style={{
         backgroundColor: "#d9d9d9",
-        padding: 20,
+        padding: 10,
+        paddingLeft: 20,
+        paddingRight: 20,
         borderRadius: 10,
         maxHeight: 250,
         width: "80%",
         overflow: "hidden",
-        marginBottom: 20,
         flexDirection: "row",
         alignItems: "center",
         justifyContent: "space-between",
-        elevation: 5,
```

Comment:
I would like to see the elevation stay so the pills looks more clickable. with it being flat, there is no real indication that the user can click on it to bring up the screen for that shot.

Reply:
I tried adding the elevation back in but the shadow didn't come back so we'll have to do more work for it to show up again. As for why I removed it, none of the other buttons in the app has a shadow or elevation of any kind so having a shadow here would be weird. If you want it to look more clickable, maybe something with the border would be better?